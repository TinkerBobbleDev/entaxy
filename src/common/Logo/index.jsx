import React from 'react'
import PropTypes from 'prop-types'
import Typography from '@material-ui/core/Typography'
import { Link } from 'react-router-dom'
import { withStyles } from '@material-ui/core/styles'
import logoImg from './logo.png'
import LinkTo from '../LinkTo'

const styles = {
  logo: {
    display: 'flex',
    minWidth: 135
  },
  image: {
    width: '30px',
    height: '30px',
    marginRight: '15px'
  },
  title: {
    textDecoration: 'none'
  }
}

const Logo = ({ classes }) => (
  <div className={classes.logo}>
    <Link to="/">
      <img
        src={logoImg}
        className={classes.image}
        alt="Entaxy logo"
      />
    </Link>
    <Typography
      variant="h6"
      color="inherit"
      className={classes.title}
      component={LinkTo('/')}
    >
      Entaxy
    </Typography>
  </div>
)

Logo.propTypes = {
  classes: PropTypes.object.isRequired
}

export default withStyles(styles)(Logo)
