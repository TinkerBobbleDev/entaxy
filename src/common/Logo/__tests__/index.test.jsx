import React from 'react'
import renderer from 'react-test-renderer'
import Logo from '../'

describe('Logo', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<Logo />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
