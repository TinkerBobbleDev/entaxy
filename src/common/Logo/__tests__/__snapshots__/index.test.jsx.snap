// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo matches snapshot 1`] = `
<img
  alt="Entaxy logo"
  className="Logo-logo-1"
  src="logo.png"
/>
`;
